bfdd/bfdd_nb_config.c

```diff
@@ -840,8 +1044,10 @@ int bfdd_bfd_sessions_single_hop_desired_echo_transmission_interval_modify(
 
 	case NB_EV_APPLY:
 		bs = nb_running_get_entry(args->dnode, NULL, true);
-		bs->peer_profile.min_echo_tx =
-			yang_dnode_get_uint32(args->dnode, NULL);
+		if (echo_interval == bs->timers.desired_min_echo_tx)
+			return NB_OK;
+
+		bs->peer_profile.min_echo_tx = echo_interval;
 		bfd_session_apply(bs);
```

Comment:
ditto:separate commit for changes related to common bfd.

Reply:
you seem to have removed those changes.
if the behavior can be kept unchanged, it is ok to me.